agent/app/agent_capability_unix.go

```diff
@@ -20,6 +20,9 @@ import (
 	"path/filepath"
 	"strings"
 
+	"github.com/aws/amazon-ecs-agent/ecs-agent/logger"
```

Comment:
nit: lets club these imports with `agent/` imports below and run goimports.

Reply:
updated this